state_machine/docs/README.md

```diff
@@ -61,4 +61,8 @@ If you make any changes to it, you can re-generate it via:
 make generate_node_state_machine_diagram
 ```
 
+
+## State Machine Lifecycle
+Node FSM starts from `stopped` state, and transitions to first `starting` and then to the `bootstapping` state, which is handled by the P2P module. In `bootstrapping` state, the node fills its address book, and upon completing mode transitions to `unsynched` state and then to `sync mode` to start requesting block metadata and receive the blocks it is missing. Upon completing syncing its state in `sync mode`; if the node is a validator node transitions to `pacemaker` state and starts participating in the consensus state, else if the node is a non-validator, node transitions to the `synced` state. However, it is important to node that a non-validator node practically always stays in the `snyc mode` to keep getting blocks from the peers. The node transitions to the `unsynched` state if it stays behind on the network, i.e. in `pacemaker` mode if the node receives a block proposal which is higher than it's current height.
```

Comment:
This is a lot of condense text that's really hard to read. I'd recommend two things:
1. User a mermaid flow chart (or link to the fsm one)
2. Use bullet points

For example:

1. **Node:** `stopped` -> `starting` -> `bootstrapping`
2. **P2P** `bootstrapping`: fill address book -> `unsynched` -> `sync mode`
3. **Consensus** `synching`: request & receiving blocks

I might be biased, but I think its easier to read. Also found some small typos. E.g. `snyc mode`